Update SecurityPage when no Security license installed

kingzacko1 opened the pull request:
## Description
<!--- Describe your changes in detail -->
Make default Graylog Security page with no license installed more informative and less like an error page. See Graylog2/graylog-plugin-enterprise#3545 for context. Open to all suggestions for changes, just getting something up for people to look at.

/nocl 

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
Closes Graylog2/graylog-plugin-enterprise#3545
## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
Purely UI change of the SecurityPage. Tested functionality of links in refactored `EnterpriseProductLink` component
## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/91903901/175132835-df9a8cb8-e2bd-4eed-850d-71afd4ee0cb0.png)

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

mikedklein commented on graylog2-web-interface/src/pages/SecurityPage.tsx:
```diff
@@ -16,60 +16,74 @@
+        <PageHeader title="Try Graylog Security">
+          {null}
+          <span>
+            Extend Graylog's capabilities for detecting, investigating, and responding to cybersecurity
```
Let's go with the review bot suggestion here it is technically a best practice albeit a nit.

mikedklein reviewed (changes requested):
I don't think the `vendor-modul-ids.json` should be in this PR right? 

The code all looks good minus the review bot suggestion and my other suggestion.

One content suggestion is to remove the graph.

<img width="851" alt="image" src="https://user-images.githubusercontent.com/3578720/175174530-d4c8f16d-7c1c-4305-be80-1ea5c25d62cf.png">

Since that is kind of redundant. That being said I am not entirely sure what to put there.

kingzacko1 commented:
@mikedklein addressed your feedback and cleaned up bot suggestion. I'm open to any ideas about what else to put on that page instead of the traffic stuff.

kingzacko1 commented:
closing. This was handled in #15300
